test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -338,10 +338,10 @@ private void NormalizationWorkout(string dataPath)
             // Apply all kinds of standard ML.NET normalization to the raw features.
             var pipeline = loader.MakeNewEstimator()
                 .Append(r => (
-                    MinMaxNormalized: r.Features.Normalize(fixZero: true),
-                    MeanVarNormalized: r.Features.NormalizeByMeanVar(fixZero: false),
+                    MinMaxNormalized: r.Features.Normalize(ensureZeroUntouched: true),
+                    MeanVarNormalized: r.Features.NormalizeByMeanVariance(ensureZeroUntouched: false),
```

Comment:
>ensureZeroUntouched [](start = 74, length = 19)

nit: ensureZeroUnchanged

Reply:
I like the word `touched`, because it makes feel that Normalizer is a cute animal which sometime plays with zeros.

---
In reply to: [265304319](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265304319) [](ancestors = 265304319)